frontend/src/views/WheresMyGene/components/HeatMap/index.tsx

```diff
@@ -255,6 +255,7 @@ export default memo(function HeatMap({
                 echartsRendererMode={echartsRendererMode}
                 setAllChartProps={setAllChartProps}
                 chartProps={allChartProps[tissue]}
+                maxExpression={6.0}
```

Comment:
can we add some comment in code why we use 6.0 here even though we're not using RankIt anymore? Thanks!

Reply:
This component is no longer used, so I didn't think any comments were necessary - when are we deleting it?

Reply:
ah I see, so 6.0 is just for passing TS. @kaloster is working on removing WMGv1!